src/components/markdown.js

```diff
@@ -4,30 +4,30 @@ import { useService } from './container-context.js';
 import ClientConfig from '../bootstrap/client-config.js';
 import GithubFlavoredMarkdown from '../common/github-flavored-markdown.js';
 
-function Markdown({ className, children, inline, renderMedia, tag, ...rest }) {
+function Markdown({ className, children, inline, tag, ...rest }) {
   const gfm = useService(GithubFlavoredMarkdown);
   const { cdnRootUrl } = useService(ClientConfig);
 
   const Tag = tag || 'div';
+  const renderMethod = inline ? 'renderInline' : 'render';
+  const innerHtml = { __html: gfm[renderMethod](children, { cdnRootUrl, renderMedia: !inline }) };
```

Comment:
I left here the only `renderMedia` switch ☝️

Reply:
:+1: